hashsign/js/src/main/scala/fluence/crypto/Utils.scala

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (C) 2017  Fluence Labs Limited
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package fluence.crypto
+
+import cats.Monad
+import cats.data.EitherT
+import io.scalajs.nodejs.buffer.Buffer
+import scodec.bits.ByteVector
+
+import scala.language.higherKinds
+import scala.scalajs.js.JSConverters._
+import scala.scalajs.js
+
+object Utils {
```

Comment:
- Please rename to smth more problem-specific. `Utils` is a too generic name.
- Docs